### DISCUSSION TITLE
Error when enabling spell checker for multiple languages

### DISCUSSION THREAD

**Discussion by ghost** [LunarVim/LunarVim#2744] [Q&A]
The first issue I found was this: https://github.com/LunarVim/LunarVim/issues/2711

When I open `nvim`and do:
1. `:set spell!`;
2. `:set spelllang=en,pt,de`.

I get the corrections and suggestions accordingly. But with LunarVim, with the same steps applied or directly in config file with `vim.opt`, it doesn't work. I get:
> Achtung: Kann Wortliste "pt.utf-8.spl" oder "pt.ascii.spl" nich finden

Basically: can't find the spell file, same as the issue mentioned; and same thing for the German spell file. Anyone can help me with this? I have the spell files in .local/nvim/site/spell, where they should be.

**Comment by martin-braun** [comment]
I have the same issue: 

<img width="683" alt="Screen Shot 2022-09-25 at 1 56 20 AM" src="https://user-images.githubusercontent.com/24259245/192122735-95c9dfe2-9b6f-45a7-9732-f8e15853f4be.png">

Can anybody help?

**Reply by JVAQUEROM** [reply]
Same problem here, any solution??
